### ISSUE TITLE
Bad dt -1 maybe with thain?

### ISSUE THREAD

**Issue by mattgodbolt** [mattgodbolt/xania#201]
```
WIZNET: [*****] BUG: Dam_message: bad dt -1.
WIZNET: RohanJunior killed by the Thain at 1111
```

**Comment by snellers** [comment]
This is due to the simulated "bleeding" that happens in update.cpp once you hit <= 0 HP.

https://github.com/mattgodbolt/xania/blob/main/src/update.cpp#L617-L619

**Comment by snellers** [comment]
Solved by de720529fe5
